Tests/test_imagefont.py

```diff
@@ -918,6 +918,46 @@ def test_cbdt_mask(self):
             assert str(e) in ("unimplemented feature", "unknown file format")
             pytest.skip("freetype compiled without libpng or unsupported")
 
+    @skip_unless_feature_version("freetype2", "2.5.1")
+    def test_sbix(self):
+        try:
+            font = ImageFont.truetype(
+                "Tests/fonts/chromacheck-sbix.woff",
+                size=300,
+                layout_engine=self.LAYOUT_ENGINE,
+            )
+
+            im = Image.new("RGB", (400, 400), "white")
+            d = ImageDraw.Draw(im)
+
+            d.text((50, 50), "\uE901", embedded_color=True, font=font)
```

Comment:
```suggestion
            d.text((50, 50), "\uE901", font=font, embedded_color=True)
```
Minor suggestion - because `font` comes before `embedded_color` in the argument list.